mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -1,14 +1,30 @@
 use futures_util::future::{self, join_all};
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use tokio::sync::mpsc::{self, UnboundedSender};
 use tracing::{error, info, info_span};
 use tracing_futures::Instrument;
 
 use crate::{bridge::Bridge, settings::BridgeSettings};
 
 /// Controller that handles the settings and monitors changes, spawns new Bridges and monitors shutdown signal.
-pub struct BridgeController {}
+pub struct BridgeController {
+    handle: BridgeControllerHandle,
+}
 
 impl BridgeController {
-    pub async fn run(system_address: String, device_id: String, settings: BridgeSettings) {
+    pub fn new() -> Self {
+        let (sender, _updates) = mpsc::unbounded_channel();
```

Comment:
What is the structure for these update publications that edgehub will send?

Reply:
will be an array of Bridge updates each with a name and an array of topic_rules